post_comment.py

```
@@ -1,111 +1,102 @@
-    - `pr_created_at`
-- `repositories`: list of repo dicts containing:
-    - `repo_owner`, `repo_name`
-    - optional `last_checked`
-- `github_access_token`: GitHub personal access token for posting comments
-
-Output keys:
-- `prs_to_review`: updated with `comment_posted=True` and cleared `files`
-- `repositories`: updated `last_checked` timestamps
+This node reads `pull_requests` and `repositories` from WaveAssist,
+posts new AI-generated comments to GitHub pull requests,
+updates each repo’s `last_checked` timestamp, and stores the updated data.
```

Ruff (0.14.10)

Docstring contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF002)